packages/storage/src/implementation/type.ts

```diff
@@ -15,6 +15,7 @@
  * limitations under the License.
  */
 
+import { isNodeSdk } from '@firebase/util';
```

Comment:
Just wondering how aggressive you want to be here in the Node environment detection. `isNodeSdk()` only checks if the Node bundle is loaded whereas `isNode()` does a runtime check to see if we are in a Node environment, regardless of what bundle was loaded. A lot of SSR cases might involve the browser bundle being loaded but trying to execute in a Node environment. I do notice this is used in the RTDB codebase and wonder if it might be worth looking at there as well.